uaclient/cli/__init__.py

```diff
@@ -67,23 +95,57 @@ def print_help_for_command(self, command: str):
         except SystemExit:
             pass
 
+    def format_help(self):
+        if self.use_main_help:
+            return super().format_help()
+        help_output = self.format_usage()
+
+        for category, items in self.help_entries.items():
+            help_output += "\n"
+            help_output += "{}:".format(category)
+            help_output += "\n"
```

Comment:
Just a nit, but this can be achieved in a single line

Reply:
I kind of like seeing the newlines on their own lines. I think it makes it a bit easier to understand what the end result will look like.

Reply:
+1 this was intentional